// 2016 misc / generator throw() caught by inner generator
module.exports = () => {
  function * generator() {
    yield * (function * () {
      try {
        yield 'foo';
      }
      catch(e) {
        return;
      }
    }());
    yield 'bar';
  }
  var iter = generator();
  iter.next();
  return iter['throw']().value === 'bar';

};
----------------------------------------------------------
Process spawn error. Is java in the path?
spawn /tmp/faast/c2c86923-2e2f-4845-9c6a-4e89ded47fe9/node_modules/google-closure-compiler-linux/compiler EACCES
